liquidator/.env.example

```
@@ -0,0 +1,13 @@
+# Testnet runtime
+LIQUIDATOR_TESTNET_RPC_URL=https://public-node.testnet.rsk.co
+LIQUIDATOR_TESTNET_FACTORY_ADDRESS=0x52C37e8364290F3A5f293D6D4ef9852B2d7D0542
+LIQUIDATOR_TESTNET_PRIVATE_KEY=0xaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaa
```

dotenv-linter (4.0.0)

[warning] [UnorderedKey] The LIQUIDATOR_TESTNET_PRIVATE_KEY key should go before the LIQUIDATOR_TESTNET_RPC_URL key

(UnorderedKey)